### DISCUSSION TITLE
Does a svg icon file exist for the app?

### DISCUSSION THREAD

**Discussion by Sunspark-007** [86Box/86Box#5049] [Q&A] [answered]
Hi, for theming purposes.. 86Box icon is in png format, but does a svg format version exist?

**Comment by lemondrops** [comment, marked as answer]
No, as it was drawn as pixel art from the start.

**Reply by Sunspark-007** [reply]
@SylEleuth did a great job here theming the icon for their [GruvBoxPlus](https://github.com/SylEleuth/gruvbox-plus-icon-pack) icon pack. I wish to thank them here for the effort it took. Thanks!

As far as I know, these are the only svg icons that exist for 86Box.

Main icon
![86Box](https://github.com/user-attachments/assets/dded2675-5dae-481d-877d-a8a3e6710d63)

Dark Symbolic
![86Box(2)](https://github.com/user-attachments/assets/f902b5b1-f59e-462c-b5f9-cf0002037446)

Light Symbolic
![86Box(1)](https://github.com/user-attachments/assets/7801bc0f-6d6d-4ca8-a073-5d28f898bfe5)
